wallet/storage/src/internal/mod.rs

```diff
@@ -118,8 +118,8 @@ impl<B: storage::Backend> WalletStorageRead for Store<B> {
 impl<B: storage::Backend> WalletStorageWrite for Store<B> {
     delegate_to_transaction! {
         fn set_storage_version(&mut self, version: u32) -> crate::Result<()>;
-        fn set_utxo(&mut self, outpoint: &AccountOutPointId, entry: Utxo) -> crate::Result<()>;
-        fn del_utxo(&mut self, outpoint: &AccountOutPointId) -> crate::Result<()>;
+        fn set_block(&mut self, block_height: &AccountBlockHeight, block: &WalletBlock) -> crate::Result<()>;
```

Comment:
This probably needs a better name. Just like `WalletBlock`.

Reply:
What should I use instead?